src/src/com/microsoft/aad/adal/AuthenticationConstants.java

```diff
@@ -349,4 +349,12 @@
     public static final int DEFAULT_EXPIRATION_TIME_SEC = 3600;
 
     public static final String AUTHENTICATION_FILE_DIRECTORY = "com.microsoft.aad.adal.authentication";
+    
+    protected static final class OAuth2ErrorCode {
```

Comment:
the only difference between protected and package visible is subclass visibility. But I agree though, this class is all about constants, cannot be subclassed anyways.